Combobox.js

```diff
@@ -539,6 +539,8 @@ define([
 								}
 							}.bind(this), 100); // worth exposing a property for the delay?
 						}
+					} else {
+						this.focusNode.focus(); // put the focus back to the inputNode.
```

Comment:
This change should be tested (that only passes after your change to Combobox.js).  Probably just updating one of the existing tests to check focus after clicking a list item.

Actually, I loaded Combobox-decl.html without your changes, and tried both the first and the second Combobox on that page, and it seems like clicking a list's item already refocuses the Combobox if autoFilter===false, or the Combobox's `<input>` if autoFilter===true??

Also, I'm confused about what happens when:

1) IIRC Combobox has a mode where there's no `<input>` visible.  What happens then?
2) Multi-select mode.   In that case don't you want to keep focus on the list item?
3) ComboPopup.   Does this code run in the ComboPopup case, and if so, doesn't focus go to the ComboPopup's `<input>` or the original ComboBox `<input>`?   And what if the ComboPopup isn't showing an `<input>`, like the first example in ComboPopup.html?